### PR TITLE
Fix typo in Delimited::open_tt

### PR DESCRIPTION
**Pull request by ia0**

### CODE REVIEW

File: src/libsyntax/ext/tt/macro_rules.rs [hunk 1]
```diff
@@ -748,7 +748,7 @@ impl TokenSet {
 }
 
 // Checks that `matcher` is internally consistent and that it
-// can legally by followed by a token N, for all N in `follow`.
+// can legally be followed by a token N, for all N in `follow`.
```
Comment (Centril) [inline src/libsyntax/ext/tt/macro_rules.rs]:
```suggestion
// can legally be followed by a token `N`, for all `N` in `follow`.
```

**Comment by JohnCSimon** [comment]
@eddyb Hello from triage - Can you please review this PR?

**Pull request merged**